docs/versionstore.md

```diff
@@ -1,3 +1,316 @@
-# Versionstore
+# VersionStore
 
-Coming soon
\ No newline at end of file
+
+(note: current doc is based on arctic-1.68.0)
+
+VersionStore serializes and stores Pandas objects, numpy arrays as well as other python types in MongoDB. Objects are `versioned` and new versions are created when a `symbol` is modified. 
+
+
+# Reading and Writing Data with VersionStore
+
+```
```

Comment:
not sure if specifying the language makes any difference, checking:

python defined:
```python 
import lz4

def print_lz4_version():
    print(lz4.__version__)
```

vs 

default lang:
```
import lz4

def print_lz4_version():
    print(lz4.__version__)
```

Reply:
basically its colors for syntax highlighting

Reply:
the dataframe output and the >>> tend to not render well when you use the `python` formatting